README.md

```diff
@@ -1,11 +1,97 @@
-# smarthome-testing
-
-## Disclaimer
-
-**This is not an officially supported Google product.**
+# Integration Testing for Smart Home Actions  
 
 ## Testing Library for Smart Home
 
-The objective is to design a scalable integration testing library for Smart Home Actions. With the enhanced logging, developers will be able to find more details about failures.
+### The problem 
+Smart home developers currently use a manual tool for testing. To solve friction in the development process, the goal is to create a way for developer's to receive more detailed information on why their errors occur.
+
+### This solution 
+The [Smart Home Integration Testing Library](https://github.com/googleinterns/smart-home-testing) introduces this solution for developers. With the enhanced logging, developers will be able to find more details about failures.
 The big impact of this will be removing friction and latency from the dev cycle, which will allow the developer to successfully verify their action more efficiently.
 
+**This repository consists of:**
+- `src/` holds the library functionalities to generate intent requests and functions to validate intent API responses against the current JSON Schemas in TypeScript.
+- `__tests__/` shows examples on how the library would interact with a developer's tests for each intent  
+- `intents/` holds the current JSON schemas for each intent response 
+- `traits/` holds the current JSON trait schemas for the on off trait, used to verify its states and attributes 
+
+## Installation 
+Use the package manager [npm](https://www.npmjs.com/) to install the packages used for the testing library 
+
+```bash
+npm install
```

Comment:
We could add a `git clone` step as well

`git clone https://github.com/googleinterns/smart-home-testing.git`
`cd smart-home-testing`
`npm install`

Reply:
Even though it's not on NPM, ideally these instructions would show them how to include this module in their project's `package.json` (without actually pulling the code into their source tree)